classes/class-gf-securesubmit.php

```diff
@@ -1833,7 +1830,7 @@ public function subscribe($feed, $submission_data, $form, $entry)
 
             $subscribResult = array(
                 'is_success' => true,
-                'subscription_id' => $plan->paymentMethodKey,
+                'subscription_id' => $planSchedule->paymentMethodKey,
```

Comment:
You may want to change the property as well to point to the schedule's id/key as well